### ISSUE TITLE
Another Stellaris issue, extra bytes at beginning or within zipfile

### ISSUE THREAD

**Issue by NonMeme** [sharkwouter/minigalaxy#616]
I get error messages pop up on Minigalaxy when I update the main Stellaris game about a .bin file being corrupted, but it doesn't affect gameplay to my knowledge.

I am trying to figure it out, and when I update a DLC pack, I don't get error messages as pop-ups. However, in Terminal, I see messages saying things like

`[/home/MY NAME/.cache/minigalaxy/download/Stellaris/Stellaris-0.bin]: [insert number here] extra bytes at beginning or within zipfile`

I substituted some words, but still, I'm thinking that this has to do with why I get error messages when updating Stellaris.

Is there a way I can remedy this? Thanks.

**Comment by sharkwouter** [comment]
That is odd. Does the same thing happen if you download and install the dlc from the GOG website?

**Comment by NonMeme** [comment]
> That is odd. Does the same thing happen if you download and install the dlc from the GOG website?

How would I view that in Terminal?

**Comment by sharkwouter** [comment]
You can run it as if it is an executable I think. I think what would be good to know is if the installer works. Sometimes GOG makes packaging errors.

**Comment by NonMeme** [comment]
> You can run it as if it is an executable I think. I think what would be good to know is if the installer works. Sometimes GOG makes packaging errors.

So, can I update it by downloading the file from Games in GOG?

**Comment by NonMeme** [comment]
I got no similar errors when using an installer from the site. But I installed the update for a DLC pack and it doesn't show as updated in Minigalaxy.

What will happen if I update that?

**Comment by sharkwouter** [comment]
Yeah, Minigalaxy has to track DLC versions manually. There is no automated way to do it.

**Comment by sharkwouter** [comment]
The `extra bytes at beginning or within zipfile` warning is actually unrelated. That happens with all Linux games and does not cause issue with extracting.

**Comment by GB609** [comment]
The zip error is no problem, corruption might rather have been related to issues with the download_manager: #246 

Or it is related to the current issues with concurrent DLC downloads for the same game:
- #579 
- #513

@NonMeme 
Stellaris has several DLC. Could you please provide info on whether you tried to install several at once (clicking their download buttons quickly after each other, without waiting for the first install to finish)?

**Comment by GB609** [comment]
This issue should be fixed since the rework of the DownloadManager.
And there was no more response to my questions for more than a year now.
